### DISCUSSION TITLE
When I use more than two cores, the following error occurs when running the code

### DISCUSSION THREAD

**Discussion by crazyfireji** [su2code/SU2#2338] [Q&A]
[i04r1n16:7873 :0:7873]    ib_iface.c:764  Assertion `gid->global.interface_id != 0' failed
==== backtrace (tid:   7878) ====
 0 0x000000000001db60 uct_ib_iface_fill_ah_attr_from_gid_lid()  /opt/hpc/software/mpi/hpcx/hpcx-v2.11-gcc-MLNX_OFED_LINUX-5-redhat7-cuda11-gdrcopy2-nccl2.11-x86_64/sources/ucx-1.13.0/src/uct/ib/base/ib_iface.c:764
![image](https://github.com/su2code/SU2/assets/112403698/e0f70402-45e5-4b37-820d-3e925fca8824)

**Comment by pcarruscag** [comment]
ib_iface.c is not an SU2 file. It may be from ParMetis, what case are you running?

**Comment by crazyfireji** [comment]
> ib_iface.c is not an SU2 file. It may be from ParMetis, what case are you running?

it's turb_ONERAM6.cfg, and the following is my script. I can run it with mpirun -n 2 SU2_CFD and -n 1 SU2, but it can't works more than 2. It's my first time using SU2.
![image](https://github.com/su2code/SU2/assets/112403698/48a8b427-570b-442c-9633-3f19b5a51311)

**Comment by bigfooted** [comment]
seems to be a file from UCX, so it's a virtual machine issue I think? If you run SU2 from inside a virtual machine, you have to make sure that the vm has access to more than 2 nodes.
